js/ionic.md

```diff
@@ -108,9 +108,7 @@ import { AmplifyService }  from 'aws-amplify-angular';
 
 export class AppComponent {
   
-  constructor( public amplify:AmplifyService ) {
```

Comment:
this would need to be defined as private in order to accessible in the other methods